src/audio/dai.c

```diff
@@ -600,6 +604,22 @@ static int dai_config_prepare(struct comp_dev *dev)
 		return -EIO;
 	}
 
+#if CONFIG_IMX8ULP
+	switch (dai->dai_index) {
+	case 5:
+		imx8ulp_dmamux_rx_int = IMX8ULP_DMAMUX2_SAI5_RX;
+		imx8ulp_dmamux_tx_int = IMX8ULP_DMAMUX2_SAI5_TX;
+		break;
+	case 6:
+		imx8ulp_dmamux_rx_int = IMX8ULP_DMAMUX2_SAI6_RX;
+		imx8ulp_dmamux_tx_int = IMX8ULP_DMAMUX2_SAI6_TX;
+		break;
+	}
+#else
+	imx8ulp_dmamux_rx_int = 0;
+	imx8ulp_dmamux_tx_int = 0;
+#endif
```

Comment:
imx8ulp_dmamux_{t|r}x_int should not be defined on other platforms. So there shouldn't be any else branch here.

Reply:
Hi Daniel, modified imx8ulp_dmamux_tx/rx only defined on imx8ulp.